msteams-platform/bots/how-to/conversations/send-proactive-messages.md

```diff
@@ -66,7 +76,6 @@ You can only install apps that are in your organizational app catalogue, or the
 See [Install apps for users](https://docs.microsoft.com/graph/teams-proactive-messaging) in the Graph documentation for complete details. There is also a [sample in .NET](https://github.com/microsoftgraph/contoso-airlines-teams-sample/blob/283523d45f5ce416111dfc34b8e49728b5012739/project/Models/GraphService.cs#L176).
 
 
-
 ## Example
```

Comment:
I'm not sure I'm following the sample code here. There should be a clear distinction between:

- A proactive message that creates a new conversation thread in a channel
- A proactive message that is a reply to an existing conversation thread in a channel
- A proactive message that is to a 1:1 or group chat

The last two might be identical, and can be combined, but the text should be clear about where/when they apply.

Reply:
Andrew, let me know when the correct code sample for Teams is available. I was just trying to "adapt" the sample from the Bot Framework. I appreciate if you can highlight the snippets I can import and the rationale behind. Thanks